src/index.ts

```diff
@@ -93,14 +93,25 @@ export class TypeScriptPlugin {
   get functions() {
     const { options } = this
     const { service } = this.serverless
+    const functions = service.functions || {}
 
-    if (options.function) {
+    const nodeFunctions = Object.keys(functions)
+      .filter(functionKey => {
+        const runtime = functions[functionKey].runtime || service.provider.runtime
+        return runtime.includes('nodejs')
+      })
+      .reduce((obj, key) => {
+        obj[key] = functions[key]
+        return obj
+    }, {})
```

Comment:
Let's not user reduce for that. It'll be clearer to setup loop as follows:

```javascript
const nodeFunctions = {};
for (const [name, functionObject] of Object.entries(functions)) {
  if (nodeRuntime) nodeFunctions[name] = functionObject;
}
```

Reply:
Good idea, I refactored the logic